index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>SalVet</title>
+    <link href="style.css" rel="stylesheet" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="top-header">
+      <i class="fa-solid fa-phone">&nbsp0757&nbsp286&nbsp893</i>
```

Comment:
Nu ar trebui să pui nimic în tagul de icoană. Iar caracterele speciale ar trebui să aibă `;` la final.
```suggestion
      <i class="fa-solid fa-phone"></i> &nbsp;0757&nbsp;286&nbsp;893
```